module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderLink.kt

```diff
@@ -1,21 +1,10 @@
 package org.cru.godtools.shared.renderer.content
 
 import androidx.compose.foundation.layout.ColumnScope
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.padding
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
 import org.cru.godtools.shared.tool.parser.model.Link
 
 @Composable
-internal fun ColumnScope.RenderLink(link: Link) = RenderLinkNode(
-    link,
-    Modifier
-        .padding(horizontal = Horizontal_Padding)
-        .fillMaxWidth(),
-)
-
-@Composable
-internal fun RenderLinkNode(link: Link, modifier: Modifier = Modifier) {
-    RenderTextNode(link.text, modifier)
+internal fun ColumnScope.RenderLink(link: Link) {
+    RenderText(link.text)
```

Comment:
sorry, what I meant by my earlier comment was that you will still call `RenderTextNode` from within `RenderLink` so you can pass a custom Modifier to it, but there was no need to introduce a `RenderLinkNode` composable between those 2

Reply:
```kotlin
fun ColumnScope.RenderLink(link: Link) {
    RenderTextNode(
        link.text,
        modifier = Modifier
            .padding(horizontal = Horizontal_Padding)
            .fillMaxWidth()
            .clickable { /* TODO */ }
    )
}
```